Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:2.7-stretch
+
+WORKDIR /app
+
+RUN groupadd --gid 10001 app \
+    && useradd -g app --uid 10001 -s /usr/sbin/nologin app
+
+
+COPY ./dev-requirements.txt /app/dev-requirements.txt
```

Comment:
Do we need this? It would make sense if we were using this docker container for local development, but I don't think we do. If we were going to use the container that way, I'd expect us to `pip install -r` this file.